### PR TITLE
M66 changelog for FIAM and ABT SDKs

### PR DESCRIPTION
**Pull request by christibbs**
Changelog for #4897.

### CODE REVIEW

File: FirebaseABTesting/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#v3.2.1
+- [added] Completion handler for FIRExperimentController's updateExperimentsWithServiceOrigin method.
+- [deprecated] FIRExperimentController updateExperimentsWithServiceOrigin:events:policy:lastStartTime:payloads:.
```
Comment (morganchen12) [inline FirebaseABTesting/CHANGELOG.md]: Repeat the verb even if it's in the tag for the sake of complete sentences. `Added completion handler ...` and `Deprecated FIRExperimentController ...`. Also, prefer Swift method syntax over objc: `FIRExperimentController.updateExperiments(serviceOrigin:events:policy:lastStartTime:payloads:)`

**Comment by christibbs** [comment]
> This isn't going out for M65 anymore - but can merge afterwards. Question though - should it be v 3.2.0?

:facepalm: yes

File: FirebaseABTesting/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# v3.2.0
+- [added] Added completion handler for FIRExperimentController's updateExperimentsWithServiceOrigin method.
+- [deprecated] Deprecated FIRExperimentController.updateExperiments(serviceOrigin:events:policy:lastStartTime:payloads:).
```
Comment (morganchen12) [inline FirebaseABTesting/CHANGELOG.md]: nit: Put the deprecated method in backticks.

**Comment by morganchen12** [comment]
@christibbs, can you update the changelog to include the added methods' names per Paul's feedback on cr/299404326?

**Comment by christibbs** [comment]
> @christibbs, can you update the changelog to include the added methods' names per Paul's feedback on cr/299404326?

https://github.com/firebase/firebase-ios-sdk/pull/5039

**Pull request merged**